code/mob_stat.dm

```diff
@@ -148,11 +148,15 @@ var/global/datum/mob_stat_thinker/mobStat = new
 		if (world.time - mobStat.last_update > mobStat.update_interval)
 			mobStat.update()
 
+		/*
```

Comment:
wait so are you deleting the map vote time in the stat panel? this is commented out but the saveStat above is not.

Reply:
No, just the part where it shows your your current vote (since I don't want to built the list every tick)